scheds/rust/scx_layered/src/main.rs

```diff
@@ -443,13 +442,15 @@ struct Opts {
     #[clap(short = 'e', long)]
     example: Option<String>,
 
-    /// Disables preemption if the weighted load fraction of a layer (load_frac_adj) exceeds the
-    /// threshold. The default is disabled (0.0).
+    /// ***DEPRECATED*** Disables preemption if the weighted load fraction
```

Comment:
These might be able to just be deleted, I don't think they are in use anywhere but I guess backwards compatibility is good. Maybe next release we can clean them out.

Reply:
> Maybe next release we can clean them out.

Idk. As a rule I want to say the cycle is:
1) Deprecate
2) Deploy
3) Delete

So if there are binaries running deployed where (1) has not happened yet, can't move to (3).